File: C++11/백준/Bronze/5235. Even Sum More Than Odd Sum/Even Sum More Than Odd Sum.cc
#include <stdio.h>

int main(){
    int a,b,c;
    scanf("%d",&a);
    for(int i=0; i<a; i++){
        scanf("%d",&b);
        int s =0;
        for(int j=0; j<b; j++){
            scanf("%d",&c);
            if(c%2==0){
                s+=c;
            }
            else{
                s-=c;
            }
        }
        if(s==0){
            printf("TIE\n");
        }
        else{
            printf("%s\n",(s>0)?"EVEN":"ODD");
        }
    }
}

File: Java/백준/Bronze/5235. Even Sum More Than Odd Sum/Even Sum More Than Odd Sum.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        for (int i =0; i<a; i++){
            int b = in.nextInt();
            int s = 0;
            for (int j=0; j<b; j++){
                int c = in.nextInt();
                if(c%2==1){
                    s+=c;
                }
                else{
                    s-=c;
                }
            }
            if(s==0){
                System.out.println("TIE");
            }
            else if(s>0){
                System.out.println("ODD");
            }
            else{
                System.out.println("EVEN");
            }
        }
    }
}